cpp/net/connection_pool.cc

```diff
@@ -32,16 +32,20 @@ using std::unique_ptr;
 using util::ClearOpenSSLErrors;
 using util::DumpOpenSSLErrorStack;
 
-DEFINE_int32(url_fetcher_max_conn_per_host_port, 4,
-             "maximum number of URL fetcher connections per host:port");
+DEFINE_int32(connection_read_timeout_seconds, 60,
+             "Connection read timeout in seconds, only applies while willing to read.");
```

Comment:
But in essence; if you're not actively trying to read or write, then it won't time out - so for example a connection idling with no requests on it wouldn't timeout.